rero/rero-ils
# public view: disable masked documents detailed

- Closes #1789.

## How to test?

- Set a document as masked and try to access it from the public view with its permalink. It should display an info message instead of the document detailed view.
